src/Hyperion/Analysis.hs

```diff
@@ -49,11 +49,12 @@ namesOf = go []
     coerce = contramap (const ()) . fmap (const ())
 
 analyze
-  :: Text -- ^ Benchmark name
+  :: Text -- ^ Package name
+  -> Text -- ^ Benchmark name
   -> Sample -- ^ Measurements
   -> Report
-analyze name samp = Report
-    { _reportBenchName = name
+analyze packageName name samp = Report
+    { _reportBenchName = Text.concat [packageName, ":", name]
```

Comment:
Has been changed.

This is needed to avoid key collisions for the json created by the benchmarks.
Indeed If I have two benches with the same name they would generate the same names, on the other hand the executable name is more likely to be unique,